spec/lib/remote_translations/caller_spec.rb

```diff
@@ -226,4 +226,67 @@
       caller.call
     end
   end
+
+  describe ".configured?" do
+    it "is true if llm? is true regardless of microsoft key" do
+      allow(RemoteTranslations::Caller).to receive(:llm?).and_return(true)
+      allow(Tenant).to receive_message_chain(:current_secrets, :microsoft_api_key).and_return(nil)
```

Comment:
```suggestion
      stub_secrets(microsoft_api_key: nil)
```

We have a helper method called `stub_secrets` to stub application secrets. We can use it here. Same for line 241.